apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -796,6 +746,84 @@ describe('/research-outputs/ route', () => {
       expect(response.status).toBe(403);
     });
 
+    test('Should return 403 when user is not permitted to publish a research output', async () => {
```

Comment:
we should say it is because you don't have a sufficient role

Reply:
we also probably want a test that is exactly like this one except we would be sending a `published: false` property and then it should be a 200